site/src/data/releases/v0.2.json

```diff
@@ -0,0 +1,160 @@
+{
+  "series": {
+    "version": "v0.2",
+    "title": "Envoy AI Gateway v0.2.x",
+    "subtitle": "Release version introducing Azure OpenAI integration, sidecar architecture, cross-backend failover, and a CLI tool.",
+    "badge": "Latest",
+    "badgeType": "milestone"
+  },
+  "releases": [
+    {
+      "version": "v0.2.0",
+      "date": "June 5, 2025",
+      "type": "minor",
+      "tags": [
+        { "text": "Azure OpenAI Integration", "type": "feature" },
+        { "text": "Sidecar Architecture", "type": "feature" },
+        { "text": "Performance Improvements", "type": "feature" },
+        { "text": "CLI Tools", "type": "feature" },
+        { "text": "Model Failover and Retry", "type": "feature" },
+        { "text": "Certificate Manager Integration", "type": "feature" }
+      ],
+      "overview": "Envoy AI Gateway v0.2.0 builds upon the solid foundation of v0.1.0 with focus on expanding provider ecosystem support, improving reliability and performance through architectural changes, and enterprise-grade authentication support for Azure OpenAI.",
+      "features": [
+        {
+          "title": "Azure OpenAI Integration",
+          "items": [
+            {
+              "title": "Full Azure OpenAI Support",
+              "description": "Complete integration with Azure OpenAI services with request/response transformation for the unified OpenAI compatiple completions API."
+            },
+            {
+              "title": "Upstream Authentication for Azure Enterprise Integration",
+              "description": "Support for accessing Azure via OIDC tokens and Entra ID for enterprise-grade authentication for secure and compliant upstream authentication."
+            },
+            {
+              "title": "Enterprise Proxy URL Support for Azure Authentication",
+              "description": "Enhanced Azure authentication with proxy URL configuration options for enterprise proxy support."
+            },
+            {
+              "title": "Flexible Token Providers",
+              "description": "Generalized token provider architecture supporting both client secret and federated token flows"
+            }
+          ]
+        },
+        {
+          "title": "Architecture Improvements",
```

Comment:
I think it is missing the important architecture improvement that we can now create multiple AIGatewayRoute and we are no longer bounded by the limits of number of rules of a single AIGatewayRoute.